build-provider.ps1

```diff
@@ -0,0 +1,56 @@
+<# Build NuGetProvider to be used immediately by PackageManagement #>
+param(
+    # Example: C:\code\OneGet
+    [Parameter(Mandatory=$true)]
+    [string]
+    $OneGetRepositoryRoot,
```

Comment:
>$OneGetRepositoryRoot [](start = 4, length = 21)

I think, this should be $PSScriptRoot, right? otherwise, curious to know in what scenarios this script file will be used.

Reply:
This is used because OneGet and NuGetProvider are in different repos. This is used to get the mt.exe location in the end.

---
In reply to: [154491954](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154491954) [](ancestors = 154491954)